lib/facilities/vha_facility.rb

```diff
@@ -44,35 +37,30 @@ def satisfaction_data(attrs)
 
       def wait_time_data(attrs)
         result = {}
-        datum = FacilityWaitTime.find(attrs['StationNumber'].upcase)
+        datum = FacilityWaitTime.find(attrs['Sta_No'].upcase)
         if datum.present?
           datum.metrics.each { |k, v| result[k.to_s] = v }
           result['effective_date'] = to_date(datum.source_updated)
         end
         result
       end
 
-      def arcgis_type
-        'VHA_Facilities'
```

Comment:
I do like that you just moved this into `pull_source_data`. No need to pollute the class.